src/Umbraco.Core/Services/FileService.cs

```diff
@@ -15,31 +15,21 @@
 
 namespace Umbraco.Core.Services
 {
-    // note
-    // file unit of work are just IUnitOfWork which is not IDisposable
-    // they don't really participate in the scope etc ;(
-    // FIXME not anymore now that Shan's made FileUnitOfWork a normal UOW? then WTF?
-
     /// <summary>
     /// Represents the File Service, which is an easy access to operations involving <see cref="IFile"/> objects like Scripts, Stylesheets and Templates
     /// </summary>
     public class FileService : ScopeRepositoryService, IFileService
     {
-        private readonly IScopeUnitOfWorkProvider _fileUowProvider;
-
         private const string PartialViewHeader = "@inherits Umbraco.Web.Mvc.UmbracoTemplatePage";
         private const string PartialViewMacroHeader = "@inherits Umbraco.Web.Macros.PartialViewMacroPage";
 
         public FileService(
-            IScopeUnitOfWorkProvider fileProvider,
-            IDatabaseUnitOfWorkProvider dataProvider,
+            IDatabaseUnitOfWorkProvider provider,

```

Comment:
Not sure we can avoid breaking things, really, but that's one of the things I want Shan to look at.